src/game/profiling.h

```diff
@@ -27,27 +33,49 @@ enum ProfilerTime {
     PROFILER_TIME_COUNT,
 };
 
+#if !PUPPYPRINT_DEBUG
+    #define PROFILER_TIME_PUPPYPRINT1 0
+    #define PROFILER_TIME_PUPPYPRINT2 0
+    #define PROFILER_DELTA_PUPPYPRINT1 0
+    #define PROFILER_DELTA_PUPPYPRINT2 0
+#endif
+
 enum ProfilerRSPTime {
     PROFILER_RSP_GFX,
     PROFILER_RSP_AUDIO,
     PROFILER_RSP_COUNT
 };
 
+enum ProfilerDeltaTime {
+    PROFILER_DELTA_COLLISION,
+#if PUPPYPRINT_DEBUG
```

Comment:
there are a lot of `#if PUPPYPRINT_DEBUG` statements in this particular file, it should be defined or not defined